.circleci/config.yml

```diff
@@ -253,6 +253,107 @@ jobs:
                 include_job_number_field: false
                 include_project_field: false
                 failure_message: ':red_circle: Scheduled tests failed on iOS device!'
+  android-build-react-native-bridge:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - checkout-submodules
+      - npm-install
+      - run:
+          name: Build JS Bundle
+          command: |
+            # Try to save memory when building the bundles by using only one
+            # worker.
+            #
+            # See:
+            #
+            # - https://stackoverflow.com/a/56027775/809944
+            # - https://facebook.github.io/metro/docs/configuration/#maxworkers
+            # - Build failure example: https://app.circleci.com/pipelines/github/wordpress-mobile/gutenberg-mobile/10073/workflows/b8eb0104-c426-4208-b077-00caac394a62/jobs/53247
+            echo "module.exports = { maxWorkers: 1 };" > ./gutenberg/packages/react-native-bridge/metro.config.js
```

Comment:
One attempt to address the out of memory failure was to tell Metro to use only one worker. It doesn't seem to work.

I tried with 2 workers as well. No luck.